tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -724,7 +728,7 @@ EOF
 		fi
 	fi
 
-	if [ "${AGENT_POLICY}" == "yes" ]; then
+	if [[ "${AGENT_POLICY}" == "yes" ]]; then
```

Comment:
nit: It's not needed to use `[[ ... ]]`, as a simple `[ ... ]` could do, and the same goes for the ones above.